File: frontend/planta-smart-4.0/src/infrastructure/dtos/response/stream/MontagemStreamDTO.ts
export interface MontagemStreamDTO {
    estacao: 'montagem';
    status: string;
    numeroOP: number;
    ocupado: boolean;
    aguardando: boolean;
    manual: boolean;
    emergencia: boolean;
    recebidoOp: boolean;
    startOP: boolean;
    finishOP: boolean;
    cancelOP: boolean;
    statusBancada: number;
    supervisorioEstoque: string;
    supervisorioProcesso: string;
    supervisorioMontagem: string;
    supervisorioExpedicao: string;
  }   

File: backend/planta-smart-4.0/src/main/java/com/smart/appsa/dto/clp/stream/MontagemStreamDTO.java
package com.smart.appsa.dto.clp.stream;

import com.smart.appsa.model.enums.StatusEstacao;

import lombok.Builder;

@Builder
public record MontagemStreamDTO(
    String estacao,
    StatusEstacao status,
    int numeroOP,
    boolean ocupado,
    boolean aguardando,
    boolean manual,
    boolean emergencia,
    boolean recebidoOp,
    boolean startOP,
    boolean finishOP,
    boolean cancelOP,
    int statusBancada,
    String supervisorioEstoque,
    String supervisorioProcesso,
    String supervisorioMontagem,
    String supervisorioExpedicao
) {}
